### PR TITLE
Switching from /user/ to /member/ API call.

### PR DESCRIPTION
**Pull request by adam-in-ict**

### CODE REVIEW

File: src/steps/access/converter.ts [hunk 1]
```diff
@@ -50,14 +50,18 @@ export function createSentryUserEntity(user: SentryUser) {
...
         username: user.email,
         role: user.role,
-        mfaEnabled: user.user.has2fa,
-        active: user.user.isActive === true,
-        isManaged: user.user.isManaged === true,
-        isStaff: user.user.isStaff === true,
-        isSuperuser: user.user.isSuperuser === true,
-        dateJoined: parseTimePropertyValue(user.user.dateJoined),
-        lastActive: parseTimePropertyValue(user.user.lastActive),
-        lastLogin: parseTimePropertyValue(user.user.lastLogin),
```
Comment (adam-in-ict) [inline src/steps/access/converter.ts:60]: The user object within user data is now an optional item with the switch to the /members/ API call.

Comment (adam-in-ict) [reply src/steps/access/converter.ts:60]: Additionally, I was going to default several of these to null, but all of them either had type restrictions due to our data model or due to existing unit tests, so I chose the below defaults as a result.

Comment (austinkelleher) [reply src/steps/access/converter.ts:60]: When will the `user` property _not_ be provided?

**Review by austinkelleher — APPROVED** [review]
Approved with question

File: src/steps/access/converter.ts (hunk 1 above)
Comment (adam-in-ict) [reply src/steps/access/converter.ts:60]: It's not provided for users who haven't accepted their invitation to join.  So we have a person with a pending invite and a person with an expired invite.  Those two will not have user.user, but all other members will.

**Pull request merged**